File: papi/lista1/exercicio2.py
numero = 0

while True:
    try:
        numero = int(input("Forneça um número: "))
        break
    except:
        print("Por favor forneça um número válido")

if numero % 2 == 0:
    print("O número é par")
else:
    print("O número é ímpar")

if numero > 0:
    print("O número é positivo")
elif numero < 0:
    print("O número é negativo")
else:
    print("O número é igual a zero")

File: papi/lista1/exercicio2.kt
package lista1

fun main() {
    var numero: Int

    while (true) {
        try {
            print("Digite um número inteiro: ")
            var input = readLine()
            numero = input!!.toInt()
            break
        } catch (e: NumberFormatException) {
            println("Por favor digite um número inteiro")
        }
    }

    if (numero % 2 == 0) {
        println("O número é par")
    } else {
        println("O número é ímpar")
    }

    if (numero == 0) {
        println("O número é igual a zero")
    } else if (numero > 0) {
        println("O número é maior que zero")
    } else {
        println("O número é menor que zero")
    }
}